### ISSUE TITLE
training on single channel pictures

### ISSUE THREAD

**Issue by njuhang** [leftthomas/SRGAN#100]
The loss function requires a vgg network, but there is no pre-trained vgg network for single-channel images. How to deal with this problem?

**Comment by leftthomas** [comment]
@njuhang solve by yourself
